docs/user-guide/building-assistants.rst

```diff
@@ -636,8 +636,7 @@ use the ``from_text`` method to extract the users whole message:
 
 .. code-block:: python
 
-    def slot_mappings(self) -> Dict[Text: Union[Dict, List[Dict]]]:
-        # type: () -> Dict[Text: Union[Dict, List[Dict]]]
+    def slot_mappings(self) -> Dict[Text, Union[Dict, List[Dict]]]:
```

Comment:
```suggestion
    def slot_mappings(self) -> Dict[Text, Union[Dict, List[Dict[Text, Any]]]]:
```
seems it's been updated even more: https://github.com/RasaHQ/rasa-sdk/blob/master/rasa_sdk/forms.py#L135